src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -273,9 +272,10 @@ private static IList<FileLocation> CreateResponseFilesList(IDictionary<string, s
 
         public static Invocation CreateInvocation(InvocationVersionOne v1Invocation,
                                            IEnumerable<NotificationVersionOne> v1ToolNotifications,
-                                           IEnumerable<NotificationVersionOne> v1ConfigurationNotifications)
+                                           IEnumerable<NotificationVersionOne> v1ConfigurationNotifications,
+                                           Run run)
```

Comment:
You know, if you're passing the run, you don't need to pass the rest of the parameters; you can get them all from the run.

Reply:
SIGH yes

Reply:
It's the way of things. As you clean code up, you discover further refinements. We're almost there now!

Reply:
Oh actually no, all that stuff is from the v1 run.

Reply:
Oh, right!